src/components/list/ListForm.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <ModalBasic
+    class="list-form"
+    name="listForm"
+    :header-text="edittingListObj ? 'edit list' : 'new list'"
+    @before-open="setListForEditting()"
```

Comment:
maybe it's gonna be more clear to put this action in some native hook of component? Created for example.